provider/resources.go

```diff
@@ -257,6 +171,29 @@ func Provider() info.Provider {
 	prov.MustComputeTokens(tfbridgetokens.SingleModule("cloudflare_", mainMod,
 		tfbridgetokens.MakeStandard(mainPkg)))
 
+	for _, r := range prov.Resources {
+		if idField, ok := r.Fields["id"]; !ok {
+			r.ComputeID = func(_ context.Context, state resource.PropertyMap) (resource.ID, error) {
+				account, hasAccount := state["accountId"]
+				zone, hasZone := state["zoneId"]
+				switch {
+				case hasAccount && hasZone:
+					return resource.ID(account.StringValue() + "__" + zone.StringValue()), nil
+				case hasAccount:
+					return resource.ID(account.StringValue()), nil
+				case hasZone:
+					return resource.ID(zone.StringValue()), nil
+				default:
+					return "id", nil
+				}
+			}
+		} else {
+			if idField.Type != "String" {
+				// TODO.
+			}
```

Comment:
What should go here?

Reply:
I'm guessing this just doesn't happen. If it did, I think that the bridge would reject the provider (so it's fine to do nothing here - we can fix it if a new release happens).